client/remote_test.go

```diff
@@ -203,7 +203,7 @@ func TestSlowServer(t *testing.T) {
 
 	// After ping checks, 1st remote must be the normal server.
 	firstRemote := g.remotes[0]
-	conn, err := firstRemote.Dial(c)
```

Comment:
Rather than assigning to `_`, this should check the error and fail the test if it's not `nil`.